packages/firestore/src/lite-api/query.ts

```diff
@@ -126,17 +126,42 @@ export function query<T>(
   return query;
 }
 
-class QueryFilterConstraint extends QueryConstraint {
+/**
+ * A `QueryFilterConstraint` is used to narrow the set of documents returned by a
+ * Firestore query by filtering on one or more document fields. `QueryFilterConstraint`s are created by invoking {@link where},
```

Comment:
ultra-nit: some of the comments in this PR are not well-formatted. e.g. this line is >120 characters, the next line is 32, and the next is 74.

I'm surprised that running `yarn prettier` in `packages/firestore` does not fix this. I know the style guide for other languages have requirements on the length of lines for readability (e.g. [C++ is max 80 chars per line](https://google.github.io/styleguide/cppguide.html#Line_Length))

I personally try to keep the comments under 80 characters per line for other languages too where I can.

Feel free to ignore ;)

Reply:
Fixed. I agree, it would be nice if the tooling formatted these comments, but it doesn't appear that is configured.